nameable.rb

```diff
@@ -0,0 +1,9 @@
+class Nameable
+  def initialize(nameable)
+    @nameable = nameable
+  end
```

Comment:
- [ ] Kindly remove this constructor. It is preventing the decorator from being tested. If you include this constructor then the `super()` method in the Person class would expect a parameter like this `super(...)`, which isn't what we want. So kindly remove this constructor.

Reply:
done